### ISSUE TITLE
Tried to create a text file in a vault

### ISSUE THREAD

**Issue by marcchataigner-zz** [consento-org/mobile#10]
I deleted 2 vaults. Then entered the 3rd one. And tried to create a text file.

First that:

![IMG_82EF10F83B55-1](https://user-images.githubusercontent.com/3974900/72858085-f5b91d80-3d02-11ea-822a-e0f292ec7c39.jpeg)

Then I `reload` and it displays that:

![IMG_7D9D14E70A04-1](https://user-images.githubusercontent.com/3974900/72858147-2dc06080-3d03-11ea-9a6b-fa48288ddf4b.jpeg)

**Comment by marcchataigner-zz** [comment]
```
Form field [fullText] already exists.
* src/util/useForm.tsx:267:22 in form.useField
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:11079:31 in mountState
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:11598:26 in HooksDispatcherOnMountInDEV.useState
* src/util/useForm.tsx:265:33 in form.useField
* src/screens/TextEditor.tsx:33:54 in Input
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:10696:27 in renderWithHooks
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:13481:6 in mountIndeterminateComponent
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:307:15 in invokeGuardedCallbackImpl
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:531:36 in invokeGuardedCallback
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:20488:8 in beginWork$$1
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:19370:24 in performUnitOfWork
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:19347:39 in workLoopSync
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:18997:22 in renderRoot
* [native code]:null in renderRoot
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:18709:28 in runRootCallback
* [native code]:null in runRootCallback
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:5642:32 in runWithPriority$argument_1
- node_modules/scheduler/cjs/scheduler.development.js:643:23 in unstable_runWithPriority
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:5638:22 in flushSyncCallbackQueueImpl
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:5627:28 in flushSyncCallbackQueue
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:18796:28 in batchedUpdates$1
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:2709:30 in batchedUpdates
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:2794:17 in batchedUpdates$argument_0
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:2814:26 in receiveEvent
- node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:436:47 in __callFunction
- node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:111:26 in __guard$argument_0
- node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:384:10 in __guard
- node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:110:17 in __guard$argument_0
* [native code]:null in callFunctionReturnFlushedQueue
```

**Comment by martinheidegger** [comment]
Closed in #a20cbfc
